.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Integration Tests
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 3 * * 1-5' # run integration tests at 3 AM, monday to friday (1-5)
+
+  workflow_dispatch: # run integration tests only when triggered manually
+
+defaults:
+  run:
+    shell: bash
+jobs:
+  integration_test:
+    name: "Integration tests"
+    runs-on: ubuntu-latest
+    env:
+      ENABLE_E2E_TEST: true
+    steps:
+      - name: Install and start K3s
+        run: curl -sfL https://get.k3s.io | K3S_KUBECONFIG_MODE="644" sh -
+
+      - name: Install Keptn
+        id: install_keptn
+        uses: keptn-sandbox/action-install-keptn@main
+        with:
+          KEPTN_VERSION: "0.13.1"
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+
+      - name: Test connection to keptn
+        run: |
+          curl -X GET "${{ steps.install_keptn.outputs.KEPTN_ENDPOINT }}/v1/metadata" -H  "accept: application/json" -H  "x-token: ${{ steps.install_keptn.outputs.KEPTN_API_TOKEN }}"
+      - name: Install Helm
+        run: curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+
+      - name: Checkout keptn-contrib/keptn-service-template-go
+        uses: actions/checkout@v3
+        with:
+          repository: keptn-sandbox/keptn-service-template-go
+          path: keptn-service-template-go
+
+      - name: Build helm chart
+        shell: bash
+        working-directory: keptn-service-template-go
+        run: gh-actions-scripts/build_helm_chart.sh 0.13.1 0.13.1 keptn-service-template-go
+
+      - name: Install helm chart
+        shell: bash
+        working-directory: keptn-service-template-go
+        run: helm upgrade --install -n keptn keptn-service-template-go installer/keptn-service-template-go-0.13.1.tgz
```

Comment:
This will only work because of some pre-defined version (e.g., 0.13.1), but not for "any branch".

Let's use the same approach as in https://github.com/keptn-contrib/job-executor-service/pull/226
Here we download artifacts (e.g., helm chart) based on the branch:

```
      # Download artifacts from last CI run
      - name: Download artifacts
        uses: dawidd6/action-download-artifact@v2.19.0
        id: download_artifacts_push
        with:
          # Download last successful artifact from a CI build
          github_token: ${{secrets.GITHUB_TOKEN}}
          workflow: CI.yml
          branch: ${{ env.BRANCH }}
          path: ./dist
```

Reply:
The CI workflow does not build a Helm chart in the `keptn-service-template-go` repository. Should I add the functionality?

Reply:
Yes, please add it!

See https://github.com/keptn-contrib/job-executor-service/blob/119264941f2081d9552a2917aba695c74f3fcccf/.github/workflows/CI.yml#L171-L197 as an example.